stats/cloud/collector.go

```diff
@@ -441,15 +443,28 @@ func (c *Collector) flushHTTPTrails() {
 	c.aggrBuckets = map[int64]aggregationBucket{}
 	c.bufferSamples = append(c.bufferSamples, newSamples...)
 }
+
+func (c *Collector) shouldStopSentMetric(err error) bool {
```

Comment:
English nitpick: would this make more sense as `shouldStopSendingMetrics` (and same for the struct field)? `stopSentMetric` sounds _off_ to me. :-/

Reply:
Ah, right, forgot changing it. Fixed.